### PR TITLE
added null mod check

### LINKED ISSUE #399: Mods sometimes don't load, and don't give friendly error
Example : misternebula/quantum-space-buddies#330

Preliminary analysis of possible causes :
- `_goHelper` is null
- `_goHelper.CreateAndAdd<IModBehaviour>` returns null

### PR DESCRIPTION
**Pull request by misternebula**
fixes #399

### CODE REVIEW

File: src/OWML.ModLoader/Owo.cs [hunk 1]
```diff
@@ -193,6 +193,17 @@ private IModBehaviour InitializeMod(Type modType, IModHelper helper)
 			{
 				var mod = _goHelper.CreateAndAdd<IModBehaviour>(modType, helper.Manifest.UniqueName);
 				_console.WriteLine("Added! Initializing...", MessageType.Debug);
+
+				if (mod == default)
```
Comment (amazingalek) [inline src/OWML.ModLoader/Owo.cs:197]: not `== null`?

**Review by amazingalek — APPROVED** [review]

File: src/OWML.ModLoader/Owo.cs (hunk 1 above)
Comment (misternebula) [reply src/OWML.ModLoader/Owo.cs:197]: default is a better choice when dealing with generics

Comment (misternebula) [reply src/OWML.ModLoader/Owo.cs:197]: the default value of any reference type is null

Comment (misternebula) [reply src/OWML.ModLoader/Owo.cs:197]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values

**Pull request merged**